Src/PerformanceCollector/Shared/Implementation/SdkVersionUtils.cs

```diff
@@ -1,16 +1,20 @@
 ﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
 {
+    using System;
     using System.Linq;
     using System.Reflection;
 
     internal class SdkVersionUtils
     {
-        internal static string GetAssemblyVersion()
+        internal static string GetSdkVersion(string versionPrefix)
```

Comment:
Yes, we have different projects and code for each is in Shared project. Shared projects do not support referencing items by link. To resolve your concern I moved this to Web.45 project and not I reference it from all other projects (like rdd.40, rdd.45 etc). That is more work but now code is in one place.

---
In reply to: [65464131](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/86#discussion_r65464131) [](ancestors = 65464131)